core/server/common/src/main/java/alluxio/master/journal/ufs/UFSJournalMultiMasterPrimarySelector.java

```diff
@@ -30,17 +31,16 @@
 import org.slf4j.LoggerFactory;
 
 import java.net.InetSocketAddress;
-import java.util.ArrayList;
-import java.util.List;
 import javax.annotation.concurrent.NotThreadSafe;
 
 /**
  * Masters use this client to elect a leader.
  */
 @NotThreadSafe
-public final class PrimarySelectorClient extends AbstractPrimarySelector
+public final class UFSJournalMultiMasterPrimarySelector extends AbstractPrimarySelector
```

Comment:
When we use UFS journal and enable ZK, this primary selector is used no matter how many masters are configured, right? So ZK enabled + 1 master -> UfsJournalMultiMaster; ZK disabled + 1 master -> UfsJournalSingleMaster?

Considering this because there's chance the user start with ZK and 1 master, and later adds masters dynamically (by starting more and not restarting the existing one).

Reply:
Ok from the Factory logic it looks good to me